### PR TITLE
Project data structure

### PR DESCRIPTION
**Pull request by alegnaaived**
Calling project from GeoServerAPI, organizing data in a reusable JSON structure for the front, fixing spanglish and sending project to the Drawer

### CODE REVIEW

File: src/Compensation.jsx [hunk 1]
```diff
@@ -33,27 +34,15 @@ class Compensation extends Component {
 
   componentDidMount() {
     Promise.all([
-      GeoServerAPI.requestProjectsGEB(),
+      GeoServerAPI.requestProjectLayersByCompany('GEB'),
       GeoServerAPI.requestBiomasSogamoso(),
       GeoServerAPI.requestSogamoso(),
+      GeoServerAPI.requestProjectNamesOrganizedByCompany('GEB'),
+      // GeoServerAPI.makeRequestTest('http://192.168.205.190:4000/geofences/ea/CORPOBOYACA'),
```
Comment (erikasv) [inline src/Compensation.jsx]: you should delete this line

File: src/compensation/Drawer.jsx [hunk 2]
```diff
@@ -82,32 +83,41 @@ class Drawer extends React.Component {
...
   }
 
+  newBiome = (layerName, szh, car, strategies) => (
+    <SelectedBiome
+      biome={layerName}
+      szh={szh}
+      car={car}
+      rows={strategies}
+      operateArea={this.operateArea}
+    // updateArea={this.}
```
Comment (erikasv) [inline src/compensation/Drawer.jsx]: you should delete this line

File: src/compensation/SelectedBiome.jsx [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+  /** tableError
+  * Set an error message above the compensations table
+  *
+  * @param {String} message message to set
+  */
+  reportTableError = (message) => {
+    this.setState({ tableError: message });
+  }
+
+  // showStrategies = (data) => {
```
Comment (erikasv) [inline src/compensation/SelectedBiome.jsx]: please avoid uploading commented code

File: .env.development.local [hunk 4]
```diff
@@ -0,0 +1,4 @@
+REACT_APP_ELASTIC_HOST='http://biotablero.humboldt.org.co/elastic-dev'
```
Comment (erikasv) [inline .env.development.local]: Not sure if there is a problem with the rule in .gitignore or how you made it it add this file, but it shouldn't be here

File: src/Compensation.jsx (hunk 1 above)
Comment (alegnaaived) [reply src/Compensation.jsx]: Done

File: src/compensation/SelectedBiome.jsx (hunk 3 above)
Comment (alegnaaived) [reply src/compensation/SelectedBiome.jsx]: Done

File: src/compensation/Drawer.jsx (hunk 2 above)
Comment (alegnaaived) [reply src/compensation/Drawer.jsx]: Done

**Pull request merged**